iterators task (1)/student/include/impl.h

```diff
@@ -5,11 +5,51 @@ template <class Container>
 struct accumulator
 {
     /** @todo Iterator traits*/
+    using iterator_category = typename std::output_iterator_tag;
+    using value_type        = Container;
```

Comment:
does it has sense?
for output iterator it's traits are usually declared as void

Reply:
that's right.